server/logo-printer.ts

```diff
@@ -0,0 +1,17 @@
+import colors = require('colors/safe');
+
+export class LogoPrinter {
```

Comment:
does not have to be a class. could just be:
```typescript
export function printAsciiLogo() {
...
}
```

Reply:
cool